### DISCUSSION TITLE
light theme for browser with limited RAM ?

### DISCUSSION THREAD

**Discussion by tebra-jl** [FreshRSS/FreshRSS#6720] [Q&A] [answered]
Hi community,
Is there a plugin or a theme that limit the memory/size of the main page of freshrss?
A good option will be to limit the number of articles to display. For example if I have 50 articles not readed but only display 10 by 10.
My goal is to read my news on the web browser of my Kindle. But it crash with big web page.
Thanks

**Comment by Alwaysin** [comment]
Have you tried https://github.com/FreshRSS/FreshRSS/pull/4353?
In ./freshrss/data/users/<user>/config.php, put icons_as_emojis as true.
Maybe also try 'show_favicons' => false, and/or simplify_over_n_feeds with your number of feed minus 1.

Just some wild guesses before someone more experienced than I am can tell you!

**Comment by Alkarex** [comment]
Good suggestions from @Alwaysin . In addition, you can:
* reduce in your preferences the number of articles shown by page
* enable in your preferences the removal of articles after reading (while waiting for https://github.com/FreshRSS/FreshRSS/issues/2347 , which you can vote on if it would help your usecase)
* disable in your preferences the autoloading of additional articles
![image](https://github.com/user-attachments/assets/c5ad09d7-4894-4b1c-9b92-ebe72a464cbb)

**Comment by tebra-jl** [comment]
Thanks both of you for your suggestions. It’s much better now.
I don’t find the “autoloading of additional articles” in my preferences.

**Reply by Alkarex** [reply, marked as answer]
I have added a screenshot to my previous response

**Comment by tebra-jl** [comment]
Don’t know why but now in portrait mode the name of the source feed isn’t displayed anymore.
![IMG_0891](https://github.com/user-attachments/assets/43cda481-ce79-4f96-b214-0f4ca813a5ea)
![IMG_0890](https://github.com/user-attachments/assets/99c47bf5-3614-4d57-ab1e-b858e7294e42)
![IMG_0889](https://github.com/user-attachments/assets/4aad5f23-6bfa-4ccc-a5af-85917dc6e0d1)
One last question, is it possible to change the font size. Don’t see any option about that.

**Reply by math-GH** [reply]
> Don’t know why but now in portrait mode the name of the source feed isn’t displayed anymore.

Have you set `show_favicons` in `config.php` from `true` to `false`?

If you did so:
The feed name will not be displayed on small screens (you named it as `portrait mode` that has a smaller pixel resolution). But the feed icons will be displayed. And you disabled it.
Would be worth to create a bug ticket. But a solution would not be simple.

**Reply by math-GH** [reply]
> One last question, is it possible to change the font size. Don’t see any option about that.

FreshRSS uses the font size defaults of your browser to calculate the icons and text sizes. The bigger/smaller the default font size is the bigger/smaller are the icons and texts in FreshRSS. You could also use the zoom feature of your browser.

If it does not work well on your eInk device browser there is one more option: Install the CustomCSS extension (see https://github.com/FreshRSS/Extensions) and set the font size to your prefered value: 

Example (set the `20px` to your needed value)
```
body {
  font-size: 20px;
}
```

**Reply by Alkarex** [reply]
P.S.: The CSS extension is now shipped by default with FreshRSS
